src/app/views/query-runner/query-input/QueryInput.styles.ts

```diff
@@ -0,0 +1,95 @@
+import { ITheme } from '@uifabric/styling';
+
+export const queryInputStyles = (theme: ITheme) => {
+  const controlWidth = '350px';
+    return {
+      autoComplete: {
+        input: {
+          minWidth: controlWidth,
+        },
+        noSuggestions: {
+          color: theme.palette.black,
+          padding: 10
+        },
+        suggestions: {
+          maxHeight: '250px',
+          overflowY: 'auto',
+          paddingLeft: 0,
+          position: 'absolute',
+          backgroundColor: theme.palette.neutralLighter,
+          minWidth: controlWidth,
+          zIndex: 1,
+          cursor: 'pointer',
+          color: theme.palette.black,
+        },
+        suggestionOption: {
+          display: 'block',
+            ':hover': {
+              background: theme.palette.neutralLight,
+            },
+          padding: 10,
+          cursor: 'pointer',
+          backgroundColor: theme.palette.white,
+          boxShadow: 'none',
+          marginTop: '0px',
+          marginRight: '0px',
+          marginBottom: '0px',
+          marginLeft: '0px',
+          paddingTop: '0px',
+          paddingRight: '32px',
+          paddingBottom: '0px',
+          paddingLeft: '12px',
+          boxSizing: 'border-box',
+          height: '32px',
+          lineHeight: '30px',
+          position: 'relative',
+          whiteSpace: 'nowrap',
+          textOverflow: 'ellipsis',
+          borderWidth: '1px',
+          borderStyle: 'solid',
+          borderColor: theme.palette.neutralLight,
+          overflow: 'hidden',
+        },
+        suggestionActive: {
+          display: 'block',
+          ':hover': {
+            background: theme.palette.neutralLight,
+          },
+          padding: 10,
+          cursor: 'pointer',
+          boxShadow: 'none',
+          marginTop: '0px',
+          marginRight: '0px',
+          marginBottom: '0px',
+          marginLeft: '0px',
+          paddingTop: '0px',
+          paddingRight: '32px',
+          paddingBottom: '0px',
+          paddingLeft: '12px',
```

Comment:
General question on GE: Is there a reason why we don't use relative units (i.e. `rem`s) instead of pixels?

Reply:
What is the benefit of relative units over pixels?

Reply:
The main concern is accessibility when the user changes their default font size in their browser. But I checked after this comment, it looks like GE doesn't use relative units in other places as well, so addressing this shouldn't be in the scope of this PR.

More reading on this:
https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css

A quick video demonstrating the accessibility benefits: https://www.youtube.com/watch?v=grMe8QREvA8